src/client/components/HealthCareApp.jsx

```diff
@@ -106,10 +106,11 @@ class HealthCareApp extends React.Component {
           throw new Error(response.statusText);
         }
         response.json().then(data => {
+          console.log(data.response);
           this.props.onUpdateSubmissionStatus('applicationSubmitted', data);
           this.props.onCompletedStatus(path);
-          this.props.onUpdateSubmissionId(data.formSubmissionId);
-          this.props.onUpdateSubmissionTimestamp(data.timeStamp);
+          this.props.onUpdateSubmissionId(data.response.formSubmissionId);
```

Comment:
Was this to fix the id not showing up on the confirmation page? Strange that timeStamp was still being displayed